lib/constants.js

```diff
@@ -0,0 +1,8 @@
+const SUPPORTED_VERBS = ['get', 'post'];
+
+const REQUEST_ID_HEADER_NAME = 'x-swatch-request-id';
```

Comment:
This feels like it should be defined in `swatchjs` itself, not in the koa binding.